tests/sentry/monitors/logic/test_mark_ok.py

```diff
@@ -13,96 +14,156 @@
     ScheduleType,
 )
 from sentry.testutils.cases import TestCase
-from sentry.testutils.helpers import with_feature
 from sentry.testutils.silo import region_silo_test
 
 
 @region_silo_test(stable=True)
 class MarkOkTestCase(TestCase):
-    @with_feature("organizations:issue-platform")
-    @patch("sentry.issues.producer.produce_occurrence_to_kafka")
```

Comment:
I've made this more unit-test like by removing the dependency on `mark_failed` from this test.